CONTRIB.md

```diff
@@ -54,25 +53,68 @@ N/A
 Aliquam in velit commodo, rutrum eros ut, condimentum nulla. Praesent tempus mi in nisi tempor posuere. Suspendisse in imperdiet justo
 ```
 
-### 2. Commits and pull requests
+### Configure Git Commit Message Template
+
+- Create a file at `~/.gitmessage-openio` (you can adapt the path if you want) with th following content:
+
+```text
+
+[TYPE]([SCOPE]): [TITLE]
+# TYPE 	CHANGE FEATURE BUGFIX/[SEVERITY] TEST REFACTOR TRIVIAL 	YES
+# SCOPE 	SDS OIOFS OIOFS_HA [COMPONENT] 	YES
+# SEVERITY 	MINOR MEDIUM MAJOR 	BUGFIX ONLY
+# COMPONENT 	only for roles: role name ; use the other values otherwise 	YES
+# TITLE 	commit title 	YES
+
+ ##### SUMMARY
+# SUMMARY 	describe what the commit does, why it is necessary, and how to use it 	YES
+
+
+ ##### IMPACT
+# IMPACT 	if the change has an impact on previous versions that can break compatibility, explain it here. therwise, set to "N/A" 	YES (N/A)
```

Comment:
```suggestion
# IMPACT 	if the change has an impact on previous versions that can break compatibility, explain it here. otherwise, set to "N/A" 	YES (N/A)
```